optimism/src/keccak/column.rs

```diff
@@ -49,12 +48,10 @@ pub enum KeccakColumn {
     PiRhoExpandRotE(usize, usize, usize),     // Round Curr[1065..1165)
     ChiShiftsB(usize, usize, usize, usize),   // Round Curr[1165..1565)
     ChiShiftsSum(usize, usize, usize, usize), // Round Curr[1565..1965)
-    IotaStateG(usize),                        // Round Next[0..100)
```

Comment:
Leaving a thought here.
Keccak is made of a non linear mapping ($\chi$), followed by a linear map ($\theta$), a transposition ($\pi$) and a translation ($\rho$), and ending with the round constants ($\iota$) - see [reference](https://keccak.team/files/Keccak-reference-3.0.pdf).
It is nearly the same structure as other zk-friendly permutations (Poseidon, Anemoi, Griffin, ...) IIUC. I wish we can find a way to uniformize it. I find it confusing to call it with Greek letters, I'm always messing up with it. If you have a way to remember the order, I would be happy to hear it.